include/range/v3/detail/config.hpp

```diff
@@ -389,6 +389,14 @@ namespace ranges
 #define RANGES_RE_ENABLE_WARNINGS
 #endif
 
+
+#if __has_attribute(no_sanitize)
+#define RANGES_INTENDED_UNSIGNED_OVERFLOW \
```

Comment:
I'm strongly opposed to any use of the word "overflow" to describe the behavior of unsigned integers. Framing the problem incorrectly creates confusion. I suggest naming this `RANGES_REALLY_UNSIGNED`.